scripts/sbml_demo.py

```diff
@@ -1,32 +1,145 @@
-from pathlib import Path
+import functools
+import logging
+import warnings
 
 import jax
 import jax.numpy as jnp
+from jaxtyping import PyTree
 
-from enzax.sbml import load_libsbml_model_from_file, sbml_to_enzax
+import equinox as eqx
+from enzax.examples.smallbone import (
+    load_smallbone,
+    get_conc_assingment_species,
+    enzax_log_density_sbml,
+)
 from enzax.steady_state import get_steady_state
+from enzax.statistical_modelling import prior_from_truth
+from enzax.mcmc import run_nuts
+
+SEED = 1234
 
 jax.config.update("jax_enable_x64", True)
 
-SBML_FILE_PATH = Path("tests") / "data" / "exampleode.xml"
+
+def get_free_params(params: PyTree) -> PyTree:
+    return (
+        params["kcat_ADH_ADH1"],
+        params["Vmax_ATPase"],
+        params["kcat_ENO_ENO1"],
+        params["kcat_ENO_ENO2"],
+        params["kcat_FBA"],
+        params["Vmax_GPD"],
+        params["kcat_GPM"],
+        params["Vmax_GPP"],
+        params["kcat_HXK_HXK1"],
+        params["kcat_HXK_HXK2"],
+        params["kcat_PDC_PDC1"],
+        params["kcat_PDC_PDC5"],
+        params["kcat_PDC_PDC6"],
+        params["kcat_PFK"],
+        params["kcat_PGI"],
+        params["kcat_PGK"],
+        params["Vmax_PGM"],
+        params["kcat_PYK_CDC19"],
+        params["kcat_TDH_TDH1"],
+        params["kcat_TDH_TDH2"],
+        params["kcat_TDH_TDH3"],
+        params["kcat_TPI"],
+        params["Vmax_TPP"],
+        params["Vmax_TPS"],
+        params["Vmax_UGP"],
+        params["Vmax_HXT"],
+    )
+
+
+def simulate(key, truth, error):
+    key_conc, key_flux = jax.random.split(key, num=2)
+    true_conc, true_flux = truth
+    conc_err, flux_err = error
+    return (
+        jnp.exp(jnp.log(true_conc) + jax.random.normal(key_conc) * conc_err),
+        true_flux + jax.random.normal(key_flux) * flux_err,
+    )
 
 
 def main():
-    model_libsbml = load_libsbml_model_from_file(SBML_FILE_PATH)
-    model, parameters = sbml_to_enzax(model_libsbml)
+    # The path should be updated.
+    file_path = (
+        "C:/Users/afjsl/Documents/enzax_clone/src/enzax/examples/"
+        "smallbone2013_model18_modified.xml"
```

Comment:
This should work:

```python
file_path = Path(__file__).parent.parent / "src" / "enzax" / "examples" / "smallbone2013_model18_modified.xml"
```
though it might be better to put the path in the `load_smallbone` function.